x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -73,6 +73,7 @@ export const GET_PROCESSES_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/running_procs`
 export const KILL_PROCESS_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/kill_process`;
 export const SUSPEND_PROCESS_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/suspend_process`;
 export const GET_FILE_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/get_file`;
+export const GET_EXECUTE_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/execute`;
```

Comment:
"get" execute?

Reply:
Yeah I thought about that too. I followed the naming pattern for GET_FILE_ROUTE. Maybe this should just be named `EXECUTE_ROUTE`?

Reply:
that's a better name.

The "get" in the previous route is not about the HTTP verb, but rather that the action is getting/fetching a file from a remote place, into the stack. To oppose a followup "send file" in releases to come

Reply:
I see. Will update it.

Reply:
done 5ace67158b5f0785a590bed546aaee9f1a55d864

Reply:
also update the PR description with action request docs